src/shared/utils/stylingFunctions.ts

```diff
@@ -0,0 +1,174 @@
+import { SettingsType } from "@src/lib/utilities/googleStorage";
+
+let messageMaxWidthStyle = "";
+let messagePaddingStyle = "";
+let messageBorderRadiusStyle = "";
+let inputBoxMaxWidthStyle = "";
+let messageBoxColors = "";
+let messageColorUserStyle = "";
+let messageColorNonUserStyle = "";
+let textColorUserStyle = "";
+let textColorNonUserStyle = "";
+const selectionColors = "";
+let messageButtonsVisibilityStyle = ``;
+const codeSnippetWidth = `
+    [data-testid^="conversation-turn-"] > * > * > *:nth-child(2) {
```

Comment:
we should start creating variables to contain the strings pointing to different points on the page and using then using them for all of these styling functions